### PR TITLE
MAISTRA-2513 more federation API refinements

### PR DESCRIPTION
**Pull request by rcernich**

### CODE REVIEW

File: federation/v1/exportedserviceset_types.go [hunk 1]
```diff
@@ -23,37 +23,37 @@ import (
 // +genclient
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
-// ServiceExport is the Schema for configuring exported services.  The name of
-// the ServiceExports resource must match the name of a MeshFederation resource
+// ExportedServiceSet is the Schema for configuring exported services.  The name of
+// the ExportedServiceSet resource must match the name of a MeshFederation resource
```
Comment (luksa) [inline federation/v1/exportedserviceset_types.go]: ...must match the name of a *ServiceMeshPeer* resource...

File: federation/v1/importedserviceset_types.go [hunk 2]
```diff
@@ -23,29 +23,29 @@ import (
 // +genclient
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
-// ServiceImports is the Schema for configuring imported services.  The name of
-// the ServiceImports resource must match the name of a MeshFederation resource
+// ImportedServiceSet is the Schema for configuring imported services.  The name of
+// the ImportedServiceSet resource must match the name of a MeshFederation resource
```
Comment (luksa) [inline federation/v1/importedserviceset_types.go]: s/MeshFederation/ServiceMeshPeer/

Comment (rcernich) [reply federation/v1/importedserviceset_types.go]: done.  also tried to remove generic "mesh" terms with "peer.:

**Comment by rcernich** [comment]
@luksa, if you're happy with this last round of changes, please give it an ack, so I can update the maistra/istio#401 PR and remove WIP.

**Review by luksa — APPROVED** [review]

**Pull request merged**